samples/constitutions/default/actions.js

```diff
@@ -959,55 +959,48 @@ const actions = new Map([
     ),
   ],
   [
-    "add_security_policy",
+    "add_snp_host_data",
     new Action(
       function (args) {
-        checkType(args.security_policy_raw, "string", "security_policy_raw");
-        checkType(
-          args.security_policy_digest,
-          "string",
-          "security_policy_digest"
-        );
+        checkType(args.security_policy, "string", "security_policy");
+        checkType(args.host_data, "string", "host_data");
       },
       function (args, proposalId) {
-        const digest = ccf.strToBuf(args.security_policy_digest);
-        const raw = ccf.jsonCompatibleToBuf(args.security_policy_raw);
+        const host_data = ccf.strToBuf(args.host_data);
+        const security_policy = ccf.jsonCompatibleToBuf(args.security_policy);
 
-        if (args.security_policy_raw != "") {
-          const redigested_raw = ccf.bufToStr(
-            ccf.digest("SHA-256", ccf.strToBuf(args.security_policy_raw))
-          );
-          const quoted_digest = ccf.bufToStr(
-            hexStrToBuf(args.security_policy_digest)
+        if (args.security_policy != "") {
+          const security_policy_digest = ccf.bufToStr(
+            ccf.digest("SHA-256", ccf.strToBuf(args.security_policy))
           );
+          const quoted_host_data = ccf.bufToStr(hexStrToBuf(args.host_data));
 
-          if (redigested_raw != quoted_digest) {
+          if (security_policy_digest != quoted_host_data) {
             throw new Error(
-              `The hash of raw policy ${raw} does not match digest ${digest}`
+              `The hash of raw policy ${security_policy_digest} does not match digest ${quoted_host_data}`
             );
           }
         }
 
-        ccf.kv["public:ccf.gov.nodes.security_policies"].set(digest, raw);
+        ccf.kv["public:ccf.gov.nodes.snp.host_data"].set(
+          host_data,
+          security_policy
+        );
 
-        // Adding a new allowed security policy changes the semantics of any other open proposals, so invalidate them to avoid confusion or malicious vote modification
+        // Adding a new allowed host data changes the semantics of any other open proposals, so invalidate them to avoid confusion or malicious vote modification
         invalidateOtherOpenProposals(proposalId);
       }
     ),
   ],
   [
-    "remove_security_policy",
+    "remove_snp_host_data",
     new Action(
       function (args) {
-        checkType(
-          args.security_policy_digest,
-          "string",
-          "security_policy_digest"
-        );
+        checkType(args.host_data, "string", "host_data");
       },
       function (args) {
-        const digest = ccf.strToBuf(args.security_policy_digest);
-        ccf.kv["public:ccf.gov.nodes.security_policies"].delete(digest);
+        const host_data = ccf.strToBuf(args.host_data);
```

Comment:
Should this be called `host_data_digest` instead to be clear that the argument is a digest?

Reply:
Host data is an abstract concept on SNP see [here](https://www.amd.com/system/files/TechDocs/56860.pdf#89) and then in confidential containers specifically it happens to be a security policy digest.

Reply:
Ah so this does not actually store the digest.